### DISCUSSION TITLE
Correction for noise in a binary system

### DISCUSSION THREAD

**Discussion by PrasannaJayanthi** [phoebe-project/phoebe2#433] [Questions]
Hello,

I was working with phoebe to produce light curves for the binary system KOI-54. I got all the other parameters to work except for the inclination. We used a inclination value of 90 and forced the eclipse but the actual value for the inclination of the system is 5. At this inclination I am getting a lot of noise in the data. Is there a step I can add to remove the noise? Also, I am getting a lot of warnings when I set the values of the parameters. I was not sure why I was getting them. 

Thank you.

**Comment by gecheline** [comment]
Hi Prasanna,

At such a low value for the inclination, you’re probably starting to see a lot of the numerical noise that goes into the horizon detection of the triangulated Phoebe mesh. One way to reduce it is to increase the number of triangles in the compute options until the numerical noise is smaller than the features in the light curve that are of interest (I doubt you will manage to completely remove it though). If the system is well detached and the stars not too distorted, you could also use some of the other backends that don’t produce triangulated meshes (legacy, ellc or jktebop).

The warnings shouldn’t be of too big of a concern if the system runs, but you can send a minimal script that reproduces your system if you want us to look into them and help you address them.

Angela
